docs/program.md

```diff
@@ -6,14 +6,114 @@ hide:
 
 ## CANOPIE-HPC 2022 Workshop Program
 Sunday, November 14th, 2022
-9am – 5:30pm CST
+8:30am – 5:00pm CST
 
 ** Note all times list CENTRAL TIME (UTC–6) **
 
 
 ## Agenda
 
-Check back later. We plan to have a combination of Invited Speakers, Research Paper Presentations and Panel sessions.
+We plan to have a combination of Invited Speakers, Research Paper Presentations and Panel sessions.
 
-|Time | Title |
-|-----|-------|
+<table>
+<thead>
+  <tr>
+    <th>Start</th>
+    <th>Duration</th>
+    <th>Title</th>
+    <th>Speaker</th>
+    <th>Affiliation</th>
+  </tr>
+</thead>
+<tbody>
+    <tr>
+        <td style="vertical-align:top" rowspan="2">08:30</td>
+        <td>5min</td>
+        <td>Workshop / Speaker Introduction</td>
+        <td>Session Shares (Shane/Andrew)</td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>30min</td>
+        <td>Keynote: Kubernetes at CERN</td>
+        <td>Richardo Rocha</td>
+        <td>CERN</td>
+    </tr>
+    <tr>
+        <td style="vertical-align:top" rowspan="3">09:00</td>
+        <td>5min</td>
+        <td>ReCap: Runtimes</td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>5min</td>
+        <td>ReCap: Image Build</td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>5min</td>
+        <td>ReCap: Scheduling</td>
+        <td></td>
+        <td></td>
+    </tr>
+    <tr>
+        <td>09:15</td>
+        <td>45min</td>
+        <td colspan="3">Paper Session #1</td>
+    </tr>
+    <tr style="background-color:#A9A9A9">
+        <td>10:00AM</td>
+        <td>30min</td>
+        <td colspan="3">Coffee Break</td>
+    </tr>
+    <tr>
+        <td rowspan="2">10:30AM</td>
+        <td>60min</td>
+        <td colspan="3">Paper Session #2</td>
+    </tr>
+    <tr>
+        <td>30min</td>
+        <td colspan="3">TBA</td>
+    </tr>
+    <tr style="background-color:#A9A9A9">
+        <td>12:00PM</td>
+        <td>90min</td>
+        <td colspan="3">Lunch Break</td>
+    </tr>
+    <tr>
+        <td rowspan="2">1:30PM</td>
+        <td>30min</td>
+        <td>Keynote: Bioinformatic workflows</td>
+        <td>Paolo Di Tommaso</td>
+        <td>Seqera Labs</td>
+    </tr>
+    <tr>
+        <td>60min</td>
+        <td colspan="3">Paper Session #3</td>
+    </tr>
+    <tr style="background-color:#A9A9A9">
+        <td>3:00PM</td>
+        <td>30min</td>
+        <td colspan="3">Coffee Break</td>
+    </tr>
+    <tr>
+        <td rowspan="3">3:30PM</td>
+        <td>30min</td>
+        <td colspan="3">TBA</td>
+    </tr>
+    <tr>
+        <td>30min</td>
+        <td colspan="3">Lighting Talks</td>
```

Comment:
Since we said we do not want to give away that we might have the option for Lighting talks... That might need to go